wireless_modules/middle_module/main.py

```diff
@@ -3,25 +3,24 @@
 import machine
 import ubinascii
 import utime
+from mqtt_client import Client
+
 
-# To get access to the 'MQTT_BROKER' variable in config.py
 try:
     import config
 except FileNotFoundError:
-    print('Error importing config.py, ensure a local version of config.py exists')
+    print("Error importing config.py, ensure a local version of config.py exists")
 
-# To get access to the Client class in the MQTT_Client_class file
-from mqtt_client import Client
 
 # Define module number and MQTT topics to publish to
-MODULE_NUM = '2'
-PUB_DATA_TOPIC = b'/v3/wireless-module/{}/data'.format(MODULE_NUM)
-PUB_TOPIC_LOW_BATTERY = b'/v3/wireless-module/{}/low-battery'.format(MODULE_NUM)
-PUB_BATTERY_TOPIC = b'/v3/wireless-module/{}/battery'.format(MODULE_NUM)
+MODULE_NUM = "2"
+PUB_DATA_TOPIC = b"/v3/wireless-module/{}/data".format(MODULE_NUM)
```

Comment:
Can you use f-strings here? e.g. `f"/v3/wireless-module/{MODULE_NUM}/data"`

We wouldn't be able to use byte strings with f strings, but I believe `paho-mqtt` is fine with passing normal strings for topics so it shouldn't be an issue.

Reply:
Sure, I'll check if normal strings are fine and if they are I'll use f-strings

Reply:
One question I have is, how does the space consumed look like? Do we or are we near the storage capacity of the MQTT?

I remember when I was doing an Arduino Uno project, and I had too much code that I had to slim down to a string format that saves the most amount of space.

Reply:
So it appears that Micropython actually doesn't support f-strings so we'll have to use .format()

Reply:
That's shame, thanks for checking though!